### DISCUSSION TITLE
Need help Lesson 5 @ 4:44:35

### DISCUSSION THREAD

**Discussion by Zetazero25** [smartcontractkit/full-blockchain-solidity-course-py#463] [Q&A] [answered]
Hi i am getting an error when i deploy the simple_storage contract

I am not sure why i am getting this error. Web3 has already been installed in my vscode 

![Screenshot 2021-11-29 211545](https://user-images.githubusercontent.com/81644486/143977048-3f273fc5-818f-40a1-a3cb-ca951e2ee866.png)

![Screenshot 2021-11-29 213407](https://user-images.githubusercontent.com/81644486/143977076-9f2d58b6-a166-49dd-a641-28b958fa82f6.png)

.

**Comment by cromewar** [comment, marked as answer]
according to this: https://github.com/smartcontractkit/full-blockchain-solidity-course-py/issues/173 this a a major brownie bug.
SO you would like to reinstall brownie using pip instead of pipx and use a virtualenv.

**Reply by Zetazero25** [reply]
the #173 link u sent, suggests I add txwait(1). I am not sure where to add that line of code. Also do i need to uninstall brownie, and then install it with pip install or can i just use pip install brownie in the terminal? i am still a beginner with this smart contract material do you have a reference link with how to use virtual env? thank you

**Reply by cromewar** [reply]
If you want con configure all your windows machine for smart contract development please refer to: https://github.com/smartcontractkit/full-blockchain-solidity-course-py#windows-support
Best regards, Cromewar
